src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -154,6 +154,20 @@ public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedCon
         }
 
         /// <summary>
+        /// Creates the silent event args list.
+        /// </summary>
+        /// <param name="suspensionContext">The suspension context.</param>
+        /// <typeparam name="T">The type of collection item.</typeparam>
+        /// <returns>The <see cref="ICollection{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
+        public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateSilentEvents<T>(this SuspensionContext<T> suspensionContext)
+        {
+            Argument.IsNotNull(nameof(suspensionContext), suspensionContext);
+            Argument.IsValid(nameof(suspensionContext.Mode), suspensionContext.Mode, mode => mode == SuspensionMode.Silent);
+
+            return new NotifyRangedCollectionChangedEventArgs[0];
```

Comment:
return ArrayShim.Empty<NotifyRangedCollectionChangedEventArgs>(); (re-uses same instance of 0 items array)

Reply:
Sure, I am not that familiar with your new class to use it "out of box". I will change.